src/routes/lobby/LobbyView.vue

```diff
@@ -179,9 +179,9 @@ watch(opponentUsername, (newVal) => {
 });
 
 // Lifecycle
-onMounted(() => {
-  playAudio(joinAudio);
-});
+// onMounted(() => {
+playAudio(joinAudio);
+// });
```

Comment:
This is a sneaky one but it needs to stay in onMounted because playAudio() is interacting with the DOM 
```suggestion
 onMounted(() => {
playAudio(joinAudio);
 });
```

Reply:
Yes, we should keep playAudio() in onMounted as this will be interacting with the browser's DOM and media APIs.
Now, I have moved back the logic inside of the onMounted() again.